src/watch.js

```diff
@@ -20,6 +20,10 @@ const assert = require('assert');
 const rbtree = require('functional-red-black-tree');
 const through = require('through2');
 
+let isPermanentError;
```

Comment:
Should these be under the "Injected" comment too, or are they somehow different?

Reply:
No longer relevant, but yes.